lib/facility.rb

```diff
@@ -1,14 +1,59 @@
 class Facility
-  attr_reader :name, :address, :phone, :services
-
-  def initialize(name, address, phone)
-    @name = name
-    @address = address
-    @phone = phone
+  attr_reader :name, :address, :phone, :registered_vehicles, :collected_fees
+  attr_accessor :hours, :services, :closed
```

Comment:
I don't believe other classes ever call to change a Facility object's `services` attribute. For that reason, I think you can keep it as an attr_reader rather than an attr_accessor.